app/views/_cascade/blocks/html/_law_footer.html

```diff
@@ -73,14 +73,70 @@
         <div class="link-container">
           <div class="links-header"><h2>Social</h2></div>
           <ul class="link-list social-icon-list">
-            <li><a class="icon icon-twitter" href="http://twitter.com/#!/chapmanu" target="_blank"><span class="sr-only">Twitter</span></a></li>
-            <li><a class="icon icon-facebook" href="http://www.facebook.com/ChapmanUniversity" target="_blank"><span class="sr-only">Facebook</span></a></li>
-            <li><a class="icon icon-pinterest" href="http://pinterest.com/chapmanu/" target="_blank"><span class="sr-only">Pinterest</span></a></li>
-            <li><a class="icon icon-graduation" href="http://itunes.apple.com/us/institution/chapman-university/id430678922" target="_blank"><span class="sr-only">iTunes</span></a></li>
-            <li><a class="icon icon-youtube" href="http://www.youtube.com/user/ChapmanUniversity" target="_blank"><span class="sr-only">youTube</span></a></li>
-            <li><a class="icon icon-linkedin2" href="http://www.linkedin.com/company/10701?trk=tyah" target="_blank"><span class="sr-only">LinkedIn</span></a></li>
-            <li><a class="icon icon-google-plus" href="https://plus.google.com/112412382370481463024/posts" target="_blank"><span class="sr-only">Google Plus</span></a></li>
-            <li><a class="icon icon-instagram" href="http://www.instagram.com/chapmanu" target="_blank"><span class="sr-only">Instagram</span></a></li>
+            <li>
```

Comment:
If you're making changes to the Law Footer block in Cascade, that file needs to be added to the corresponding `.cascade-code/` directory, with the added changes.

If not, this HTML view shouldn't be edited.

Reply:
@mandythomas I've removed all `span` tags and included `title` tags within all of the SVG's.